app/jinja_filters.py

```diff
@@ -66,8 +67,11 @@ def format_multilined_string(context, value):
 
 
 @blueprint.app_template_filter()
-def format_date(value, date_format='%-d %B %Y'):
-    return "<span class='date'>{date}</span>".format(date=datetime.strptime(value, '%Y-%m-%d').strftime(date_format))
+def format_date(value, date_format='%B %Y'):
+    if value and re.match(r'\d{4}-\d{2}-\d{2}', value):
```

Comment:
Do we need any of this logic here if it is done in `convert_to_datetime`?

Reply:
Here it is used to just define the `date_format`, the other ones is used to convert to a datetime object aswell. The `date_format` differs between the two.

Reply:
Ah yes, same logic but different formats